MediaBrowser.Common/Json/Converters/JsonCommaDelimitedArrayConverter.cs

```diff
@@ -22,11 +22,17 @@ public JsonCommaDelimitedArrayConverter()
         }
 
         /// <inheritdoc />
-        public override T[] Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        public override T[]? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
         {
+            if (reader.TokenType == JsonTokenType.Null)
+            {
+                return null;
```

Comment:
```suggestion
                return Array.Empty<T>();
```

Reply:
Would break the `JsonCommaDelimitedArrayTests.Deserialize_String_Null_Success` test

Reply:
Either the test needs to change, or JsonPipeDelimitedArrayConverter needs to change as it returns `Array.Empty` when given a null value

Reply:
Looks like it never hits this block and I don't know why

Reply:
Interesting, I just tested it and also found that this block is never hit. Either way, I think that `JsonCommaDelimitedArrayConverter` and `JsonPipeDelimitedArrayConverter` should behave the same, whichever way we go in the end